packages/svelte-flicking/src/flicking-panel.svelte

```diff
@@ -3,9 +3,9 @@
     onDestroy,
     getContext
   } from "svelte";
-  import * as uuid from "uuid-browser";
+  import { v4 } from "uuid";
```

Comment:
Why did you change from uuid-browser to uuid?

Reply:
I believe that was because `uuid-browser` couldn't run in test env.
While the `uuid` can run in both env.

Reply:
uuid contains binary modules. It will be difficult to work in the browser.

Reply:
![image](https://user-images.githubusercontent.com/3433062/142371236-91a596ec-3c26-457a-a07c-f76f908cfb8e.png)

Reply:
Well if you're using the esm module just with CDN like that, you're right.
But in svelte we're using it with bundlers by default, aren't we?
They even say they can use it in the browsers.
- https://www.npmjs.com/package/uuid#user-content-ecmascript-modules

Also, I've tested in both my local dev-env & other env that use standard svelte dev-env.